media-libs/openjpeg/files/openjpeg-2.4.0-fix-segfault.patch

```diff
@@ -0,0 +1,23 @@
+From 0afbdcf3e6d0d2bd2e16a0c4d513ee3cf86e460d Mon Sep 17 00:00:00 2001
+From: xiaoxiaoafeifei <lliangliang2007@163.com>
+Date: Wed, 14 Jul 2021 09:35:13 +0800
+Subject: [PATCH] Fix segfault in src/bin/jp2/opj_decompress.c due to
+ uninitialized pointer (fixes #1368) (#1369)
+
+---
+ src/bin/jp2/opj_decompress.c | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/bin/jp2/opj_decompress.c b/src/bin/jp2/opj_decompress.c
+index 0e0287350..18ead6721 100644
```

Comment:
Normally, I would scrub a patch to remove some cruft, but as I am not the author of the patch, I figured it's maybe best to leave it as-is. Let me know if I should scrub it.

Reply:
Scrubbing won't remove From or Subject, just git noise, so it should be fine to scrub.

Reply:
Scrubbed!

Some output from scrub-patch. I'll add a link.
```
QA: openjpeg-2.4.0-r2-fix-segfault.patch has generated warnings:

001: From 0afbdcf3e6d0d2bd2e16a0c4d513ee3cf86e460d Mon Sep 17 00:00:00 2001
^^^ suggest replacing line with a http link pointing the upstream commit hash
```